checker/src/org/checkerframework/checker/igj/IGJAnnotatedTypeFactory.java

```diff
@@ -27,6 +28,7 @@
 import org.checkerframework.framework.util.AnnotatedTypes;
 import org.checkerframework.framework.util.GraphQualifierHierarchy;
 import org.checkerframework.framework.util.MultiGraphQualifierHierarchy.MultiGraphFactory;
+import org.checkerframework.framework.util.defaults.QualifierDefaults;
```

Comment:
These two added imports seem unnecessary, as no code was added.
Does your IDE provide "unused imports" warnings? Then the IDE would warn you about such unnecessary code.